core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -3103,12 +3106,37 @@ public function initPicker(PickerInterface $picker)
 				throw new \RuntimeException('Invalid path ' . $strPath);
 			}
 
+			if (empty($this->arrFilemounts) && null === ($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['root'] ?? null))
+			{
+				$this->arrFilemounts = array($strPath);
+			}
+
+			foreach ($this->arrFilemounts as $i => $strFolder)
```

Comment:
@ausi This could be wrapped in an `else` block, couldn‘t it?

Reply:
Yes.

Reply:
Changed in 9571d203488026494691d53ccf100d451855dfbc.